tests/unit/test_zarrio.py

```diff
@@ -122,3 +129,24 @@ class TestExportZarrToZarrNestedDirectoryStore(BaseTestExportZarrToZarr):
     def setUp(self):
         super().setUp()
         self.store = [NestedDirectoryStore(p) for p in self.store_path]
+
+
+#########################################
+#  Consolidate Metadata tests
+#########################################
+class TestConsolidateMetadata(TestCase):
+    """
+    Tests for consolidated metadata and corresponding helper methods.
+    """
+    def test_get_store_path_shallow(self):
```

Comment:
Should there be a ``def tearDown()`` method here to delete the `'tests/unit/example.zarr'` and `'tests/unit/test_consolidate.zarr'` files? And related to that, are those two files required to be stored in the repo or should they be removed from the PR? It looks like the files are automatically generated each time the test-suite is run, so it seems they are probably not needed.

Reply:
I actually wanted to keep them, similar to how we keep the termset example for hdmf. It seemed easier to have a set of test files I could always provide a path to for future tests that can be in different test scripts.

Reply:
The key difference is that TermSet are not generated or modified by the tests but only used. Here, however, the files are a main component of the test, which means: 

- if a test modifies the files then that leads to many updates and cross-talk between tests that are hard to debug.
- if we need to update the files in the future, there is no record how these files where generated

As an alternative, I think it may be more robust to add a function to  `https://github.com/hdmf-dev/hdmf-zarr/blob/dev/tests/unit/utils.py` to create (and delete) the test files so that it can be easily generated and cleaned-up by the tests that need a basic test file. In this way it is also explicit how these files are being generated.

Reply:
Sounds good.